benefits/untracked_holiday.md

```diff
@@ -4,6 +4,10 @@ We're building a high-performance organisation that delivers great software prod
 
 We have adopted an untracked holiday policy, where each team member is given the flexibility to take whatever holiday they need. We believe this policy will help us to perform at the highest possible level. 
 
+It is important that you and your colleagues feel as though that holiday is highly available. That holiday is an easily accessible commodity that enables everyone to get the necessary rest that they need. We intend untracked holiday to prevent burn-out in situations, where individuals feel that they need to "save up" holiday days, or plan elaborate strategies around using your holiday allowance "effectively" later in the year, rather than taking holiday when needed most.
+
+Untracked Holiday is not meant as a way to regularly take large portions of time off work e.g. 1 week every month, or 4 months off straight. Please see our [Flexible Working](flexible_working.md) as the alternative to Untracked Holiday in those situations.
```

Comment:
Since we generally work in pairs, if I was to observe someone being regularly unavailable within my delivery team. I would feel frustrated and disrespected by them. As it would impact my need to deliver value to our customers and my need to ensure my pair could take over from me if I needed to take holiday.

I'm not sure what the number is exactly, but those needs should be met IMHO